packages/user-group-service/src/users/typedef.graphql

```diff
@@ -109,8 +16,10 @@ enum ldapFieldType {
 
 type RoverUserType {
   uid: String
+  name: String @deprecated(reason: "Field is deprecated!")
   cn: String
   rhatJobTitle: String
+  title: String @deprecated(reason: "Field is deprecated!")
```

Comment:
Should be in the `UserType`. Can we remove `RoverUserType`, and just use `UserType`? It's confusing to have 2 different gql types for user, and I don't think it is needed.

Just mark the old fields as `@deprecated`, and the new fields from rover to `UserType`

Reply:
@deshmukhmayur that's for the cases when somebody wants to fetch users directly from Rover for additional fields like `rhatGeo`, & others i.e. not defined in the `UserType`.